.github/workflows/performance-tests.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: Run performance tests using latest published docker image
+on:
+  # to iterate on during PR
+  pull_request: {}
+  schedule:
+    - cron: '0 2 * * 0-4'
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.PERFORMANCE_AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.PERFORMANCE_AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: us-west-2
```

Comment:
How do you feel about a less-used region like us-east-2? Or are you using a different account than the main sandbox?

Reply:
Sure, I'll try deploying to us-east-2